opp_msgs_serialization/include/opp_msgs_serialization/opp_msgs_yaml.h

```diff
@@ -151,6 +149,37 @@ struct convert<noether_msgs::PlaneSlicerRasterGeneratorConfig>
   }
 };
 
+template <>
+struct convert<heat_msgs::HeatRasterGeneratorConfig>
+{
+  static Node encode(const heat_msgs::HeatRasterGeneratorConfig& rhs)
+  {
+    Node node;
+
+    node["point_spacing"] = rhs.point_spacing;
+    node["raster_spacing"] = rhs.raster_spacing;
+    node["tool_offset"] = rhs.tool_offset;
+    node["min_hole_size"] = rhs.min_hole_size;
+    node["min_segment_size"] = rhs.min_segment_size;
+    node["raster_rot_offset"] = rhs.raster_rot_offset;
+    node["generate_extra_rasters"] = rhs.generate_extra_rasters;
+    return node;
+  }
+
+  static bool decode(const Node& node, heat_msgs::HeatRasterGeneratorConfig& rhs)
+  {
+    if (node.size() != 7)
+      return false;
+    rhs.raster_spacing = node["raster_spacing"].as<decltype(rhs.raster_spacing)>();
```

Comment:
Looks like you forgot `point_spacing`

```suggestion
    rhs.point_spacing = node["point_spacing"].as<decltype(rhs.point_spacing)>();
    rhs.raster_spacing = node["raster_spacing"].as<decltype(rhs.raster_spacing)>();
```